cmd/containerd-shim-runc-v2/process/io.go

```diff
@@ -107,6 +110,16 @@ func createIO(ctx context.Context, id string, ioUID, ioGID int, stdio stdio.Stdi
 		pio.io, err = runc.NewPipeIO(ioUID, ioGID, withConditionalIO(stdio))
 	case "binary":
 		pio.io, err = NewBinaryIO(ctx, id, u)
+	case "attachablebinary":
```

Comment:
Needs godoc and test

Reply:
test added.
I am not really sure where to add godoc cc @AkihiroSuda

Reply:
godoc added

Reply:
maybe we can merge `attachablebinary` with existing one, `binary`, by using reserve param key like `_attach=true`.

cc @AkihiroSuda @dmcgowan

Reply:
I intended to merge them but I think it will introduce a non-negligible complexity and more responsibilities to `NewBinaryIO()` func so I prefer to make a seperate function

Reply:
> maybe we can merge attachablebinary with existing one, binary, by using reserve param key like _attach=true.

That was my thinking too. "attachablebinary" is extended case of "binary", I like the idea of introducing the flag. The code is indeed complex and needs some refactoring.